buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -0,0 +1,29 @@
+object VersionsBackend {
+    const val zxing = "3.5.0"
+    const val semver = "1.1.2"
+    const val `google-cbor` = "0.01.02"
+    const val hsmf = "0.8.3"
+    const val pgsql = "42.7.3"
+    const val mockito = "5.1.0"
+    const val okhttp = "4.11.0"
+    const val scrimage = "4.0.39"
+    const val pupilidlib = "1.9.0-pupilid"
+    const val attestation = "2.1.3"
+    const val jose = "9.25.6"
+
+    object spring {
```

Comment:
why do we need spring dependency declarations here?

Reply:
removed